src/index.js

```diff
@@ -51,14 +58,15 @@ function pingServers () {
  * Pings and individual server to update the arp table.
  */
 function pingServer (address) {
-  return new Promise(function (resolve, reject) {
-    new net.Socket()
-      .setTimeout(1000, close)
-      .connect(80, address, close)
-      .once('error', close)
+  return new Promise(function (resolve) {
+    // https://stackoverflow.com/a/51986121/1238150
```

Comment:
I don't think that is relevant here unless I'm missing something?

Reply:
Nope, it is not really relevant. Sometimes, I just add reference links for myself. If I ever stumble upon this line and wonder why I added it, I can take a look again. But in this case, it was just for me and I'll remove it.